### PR TITLE
4472 4473 last updated fix

### PR DESCRIPTION
**Pull request by hyi**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Comment by hyi** [comment]
This PR will fix both issues #4472 and #4473, so last updated time stamp for resources affected by resource versioning fix will be set back it their old time stamp and the replaced resources will maintain their time stamp rather than replaced with their versioned resources' time stamps. @pkdash I assigned you as a reviewer.

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4487](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4487)

File: hs_core/management/commands/fix_resource_versioning_date.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            date_col = 1
+            for row in reader:
+                res_id = row[res_col]
+                res_date = row[date_col]
+                try:
+                    res = get_resource_by_shortkey(res_id)
+                except Exception:
+                    print(f'Resource {res_id} does not exist', flush=True)
+                    continue
+                res.updated = res_date
```
Comment (pkdash) [inline hs_core/management/commands/fix_resource_versioning_date.py:28]: I think we also need to update the `Date` metadata for date type `modified`.

**Review by pkdash — CHANGES_REQUESTED** [review]

**Comment by hyi** [comment]
@pkdash I have addressed all of your comments. See if they look good now. Thanks!

File: hs_core/management/commands/fix_resource_versioning_date.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import csv
 from django.core.management.base import BaseCommand
 from hs_core.hydroshare.utils import get_resource_by_shortkey
+from hs_core.models import Date
```
Comment (pkdash) [inline hs_core/management/commands/fix_resource_versioning_date.py:4]: Not needed

**Review by pkdash — CHANGES_REQUESTED** [review]
Left a minor change suggestion.

**Review by pkdash — APPROVED** [review]

**Pull request merged**